docs/docs/documentation/user-guide/create-agents.md

```diff
@@ -544,19 +549,44 @@ This gives your agent multiple greeting styles to choose from based on the conte
 
 ## Running Your Agent
 
-To run a plugin agent, you first need to package and install it as a plugin. But for debugging or isolated development testing, you can run your agent from the `src` directory directly using the SAM CLI.
+To run a plugin agent, you first need to package and install it as a plugin. 
+
+:::tip[Quick Debug]
+
+For debugging or isolated development testing, you can run your agent from the `src` directory directly using the SAM CLI.
 
 ```bash
 cd src
 sam run ../config.yaml
 ```
 
 By moving to the `src` directory, the module path is set correctly, allowing SAM to find your functions without having to install them in your python environment as a plugin package.
```

Comment:
Changing to the `src` directory allows the module path to be set correctly so that SAM can find your functions without your having to install them in your python environment as a plugin package.